src/auto_code_surveys.py

```diff
@@ -14,7 +14,7 @@
 
 class AutoCodeSurveys(object):
     SENT_ON_KEY = "sent_on"
-    ICR_MESSAGES_COUNT = 200
+    ICR_MESSAGES_COUNT = 10000
```

Comment:
Why is this number so high?

Reply:
I'll need to write a separate export to output all messages for thematic analysis. We probably won't do ICR.

Reply:
Updated now

Reply:
(if you fixed something in a future PR it would be helpful to say that in your comment here so that I know where to look. Was easy to find myself in this case though).

Reply:
Noted!